templates/footer.hbs

```diff
@@ -23,7 +23,7 @@
         </address>
       </div>
     <ul class="footer-nav">
-      <li><a href={{settings.mit_home}}>MITxPro Home</a></li>
+      <li><a href={{settings.mit_home}}>MIT xPro Home</a></li>
```

Comment:
Please correct me if I'm wrong, as per Peter's comments isn't it supposed to be `MIT xPRO` with capital PRO?

Reply:
yes.